tests/test_on_data/ophys/test_imaging_interfaces.py

```diff
@@ -1123,3 +1124,356 @@ def check_extracted_metadata(self, metadata: dict):
+        munit_name="MUnit_0",
+        channel_name="UG",
+    )
+    save_directory = OUTPUT_PATH
+
+    def check_extracted_metadata(self, metadata: dict):
+        """Check that the metadata was extracted correctly for p30.mesc."""
+
+        # Check session start time - different from p29
+        assert metadata["NWBFile"]["session_start_time"] == datetime(2017, 9, 30, 9, 36, 12, 98727, tzinfo=timezone.utc)
+
+        # Check NWBFile metadata
+        nwbfile_metadata = metadata["NWBFile"]
+        assert nwbfile_metadata["session_description"] == "Session: MSession_0, MUnit: MUnit_0."
+        assert nwbfile_metadata["experimenter"] == ["flaviod"]
+        assert nwbfile_metadata["session_id"] == "071c1b91-a68a-46b3-8702-b619b1bdb49b"
+
+        # Check device metadata
+        device_metadata = metadata["Ophys"]["Device"][0]
+        assert (
+            device_metadata["name"] == "Microscope"
+        )  # Default metadata because this was not included in the source metadata
+        assert device_metadata["description"] == "version: MESc 3.3, revision: 4356"
+
+        # Check imaging plane metadata
+        imaging_plane = metadata["Ophys"]["ImagingPlane"][0]
+        assert imaging_plane["name"] == "ImagingPlane"
+        assert (
+            imaging_plane["device"] == "Microscope"
+        )  # Default metadata because this was not included in the source metadata
+        assert imaging_plane["imaging_rate"] == 30.962890625
+        assert imaging_plane["grid_spacing"] == [1.7821140546875, 1.7821140546875]
+        assert imaging_plane["grid_spacing_unit"] == "µm"
+
+        # Check optical channel metadata
+        optical_channel = imaging_plane["optical_channel"][0]
+        assert optical_channel["name"] == "UG"
+        assert (
+            optical_channel["description"]
+            == "An optical channel of the microscope. PMT voltage: 65.0V, Warmup time: -0.2s"
+        )
+
+        # Check two photon series metadata
+        two_photon_series = metadata["Ophys"]["TwoPhotonSeries"][0]
+        assert two_photon_series["name"] == "TwoPhotonSeries"
+        assert (
+            two_photon_series["description"] == "Imaging data from two-photon excitation microscopy."
+        )  # Default metadata because this was not included in the source metadata
+        assert (
+            two_photon_series["unit"] == "n.a."
+        )  # Default metadata because this was not included in the source metadata
+        assert two_photon_series["dimension"] == [512, 512]
+
+        # Check geometric transformations in imaging plane description
+        assert (
+            imaging_plane["description"]
+            == "The plane or volume being imaged by the microscope. Geometric transformations: translation: [  -456.221198   -456.221198 -11425.51    ], rotation: [0. 0. 0. 1.], labeling_origin: [     0.        0.   -11281.89]"
+        )
+
+
+# class TestFemtonicsImagingInterfaceSingleChannel(ImagingExtractorInterfaceTestMixin):
```

Comment:
Ok, these are for the incoming tests. Did you test them locally? I am still unable to make gin work.